src/libs/OptionsListUtils.js

```diff
@@ -796,6 +870,15 @@ function getCategoryListSections(categories, recentlyUsedCategories, selectedOpt
         indexOffset += selectedOptions.length;
     }
 
+    const selectedOptionNames = _.map(selectedOptions, (selectedOption) => selectedOption.name);
+    const filteredRecentlyUsedCategories = _.chain(recentlyUsedCategories)
+        .filter((categoryName) => !_.includes(selectedOptionNames, categoryName) && lodashGet(categories, `${categoryName}.enabled`, false))
```

Comment:
Maybe let's use array-based `lodashGet` here, as we modify this code (also a few lines later)

Reply:
I am not sure I understand what you want here. 
Should we use `lodashGet` instead of `includes`? Actually, `selectedOptions` and `selectedOptionNames` are both arrays, therefore `lodashGet` is not a good choice to iterate here.

Reply:
```
lodashGet(categories, `${categoryName}.enabled`)
```

It's just a follow-up to our earlier conclusions about using `lodashGet` with a dot-based string argument. The variant which accepts an array is safer when the strings are dynamic.